src/app/qgsdevtoolspanelwidget.cpp

```diff
@@ -103,3 +124,9 @@ void QgsDevToolsPanelWidget::setCurrentTool( int row )
   whileBlocking( mOptionsListWidget )->setCurrentRow( row );
   mStackedWidget->setCurrentIndex( row );
 }
+
+void QgsDevToolsPanelWidget::showUrl( const QUrl &url )
+{
+  whileBlocking( mOptionsListWidget )->setCurrentRow( 0 );
```

Comment:
This is a little fragile, eg if we move around the rows later. I think instead you should call setActiveTab here.